src/pages/javascript/switch-statements/index.md

```diff
@@ -2,14 +2,71 @@
 title: Switch Statements
 ---
 ## Switch Statements
+<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A `switch` statement in programming is similar to an `if-else` statement, but has the benefit of sometimes being easier to read when there are a lot of conditions. Also it allows a `default` block to be added that will be executed if none of the other conditions are true.
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/switch-statements/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### Syntax:
+```javascript
+switch(expression) {
+    case 1:
+        console.log('1');
+        break;
+    case 2:
+        console.log('2');
+        break;
+    default:
+        console.log('No true condition, default');
+}
+```
```

Comment:
Sorry I didn't catch this yesterday 😅 - looks like freecodecamp.org uses 2 spaces to indent JS code. I think we should update these code blocks to be consistent with that for Guide coding examples.

Reply:
That's fine :) Fixed now!